includes/SemanticData.php

```diff
@@ -200,6 +200,15 @@ public function __sleep() {
 		return [ 'mSubject', 'mPropVals', 'mProperties', 'subSemanticData', 'mHasVisibleProps', 'mHasVisibleSpecs', 'options', 'extensionData', 'sequenceMap' ];
 	}
 
+	/**
+	 * @since 3.2
+	 *
+	 * @return boolean
+	 */
+	public function isStub() {
```

Comment:
Why not use return types in new code now that we can?

Reply:
Well, I forget about that on this PR.

Reply:
Shit, I got competition in the forgetting department it seems ;)